src/net/posix-stack.cc

```diff
@@ -960,10 +960,10 @@ std::vector<network_interface> posix_network_stack::network_interfaces() {
                     rtgenmsg gen;
                     ifaddrmsg addr; 
                 }; 
-            } req = { {0}, };
+            } req = {};
 
-            sockaddr_nl kernel = { 0, }; 
-            msghdr rtnl_msg = { 0, };
+            sockaddr_nl kernel = {};
```

Comment:
Note, that this sockaddr_nl is initialized by the kernel, so zeroifying it makes no practical sense

Reply:
i agree, but i'd prefer leave this to another change. to avoid initializing fields in an output parameter is not in the scope of this PR. it created to address the compiling warning. we can have another PR to do this. and we would need to test it. again, i don't have the environment for testing DPDK application. FWIW, that's also one of reasons why https://github.com/scylladb/seastar/pull/1528 is stalled.

Reply:
switched to designated initializer anyway. i still agree with you that zerofying it does not make sense. but i would leave it to another change.